src/SFA.DAS.Apprenticeships.Web.UnitTests/Controllers/AllControllersTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.Reflection;
+
+namespace SFA.DAS.Apprenticeships.Web.UnitTests.Controllers
+{
+	public class AllControllersTests
+	{
+		/// <summary>
+		/// This Test ensures that all controllers have the ValidateAntiForgeryToken attribute on all HttpPost, HttpPut and HttpDelete methods
+		/// </summary>
+		[Test]
+		public void VerifyAntiForgeryTokenAttributeIsPresent()
```

Comment:
Probably don't need to validate that post requests explicitly have antiforgery attributes as this has been setup with autovalidate in program.cs